rotkehlchen/exchanges/kraken.py

```diff
@@ -513,14 +516,14 @@ def query_balances(self) -> Tuple[Optional[dict], str]:
                 log.error(msg)
                 return None, msg
 
-        balances = {}
-        for k, v in old_balances.items():
-            v = FVal(v)
-            if v == FVal(0):
+        asset_balance: DefaultDict[Asset, Balance] = defaultdict(Balance)
+        for kraken_name, amount_ in old_balances.items():
```

Comment:
can you rename old_balances to kraken_balances? Kind of a stupid name we had there before :)